Teamkk

kerryhuang1 opened the pull request:
Used react-schedule-selector to make the dragging functionality. Ultimately decided to just implement a weekly view. Still fleshing out some details.

bdeleonardis1 commented on client/src/components/Calendar/TODO.txt:
```diff
@@ -0,0 +1,6 @@
+EVERYTHING THAT STILL NEEDS TO BE DONE:
```
Like I mentioned for the other thing, let's just file github issues for all these and delete the file.

bdeleonardis1 commented on client/src/index.js:
```diff
@@ -1,7 +1,14 @@
+import Calendar from "./components/Calendar/Calendar";
 
-import App from "./App.js"
+function App() {
```
Let's revert this to the old index.js file. You already have the calendar display at the /calendar link, no reason to do it here too, right?

kerryhuang1 commented on client/src/App.js:
```diff
@@ -3,24 +3,24 @@ import { BrowserRouter, Switch, Route } from 'react-router-dom';
   constructor(props) {
     super(props);
   }
-
```
Not sure what this means

kerryhuang1 commented:
RAFL @bdeleonardis1 @leebianca 

Summary: 
- Removed package2.json, TODO.txt 
- Reverted index.js to starter code
- Reformatted with prettier
- Removed all comments
